### PR TITLE
Initialize Redux

### PR DESCRIPTION
**Pull request by simplegoose**
- Create a redux directory
- Create book and category slices

### CODE REVIEW

File: src/redux/store.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { configureStore } from '@reduxjs/toolkit';
+
+const store = configureStore({
+  reducer: {
+
+  },
+});
+
```
Comment (AmaduKamara) [inline src/redux/store.js]: - [x] After creating your reducers, you're expected to import them in this file and use them inside the reducer object with their key and value pairs. 

`reducer: { books: booksReducer, categories: categoriesReducer }`

- [x] Kindly import the bookReducer and the categoriesReducer and make use of them inside the reducer object in your store 👍🏽

File: src/redux/categories/categorySlice.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+const initialState = {
+  categories: [],
+};
+
+const categorySlice = createSlice({
+  name: 'category',
+  initialState,
+  reducers: {
+    checkStatus: () => ({ categories: 'Under construction' }),
+  },
```
Comment (AmaduKamara) [inline src/redux/categories/categorySlice.js]: - [x] The checkStatus action is expected to take the state as an argument and which is then get used for updating that state. Kindly pass the state into your reducer action 👍🏽

```javascript
checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
```

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi @simplegoose,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ Used redux toolkit and created reducers and actions 👍🏽
✅ App is structured well using the ducks pattern 👍🏽

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by gemmen29 — APPROVED** [review]
Hi @simplegoose, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**